README.rst

```diff
@@ -195,7 +195,7 @@ the `low-level API`__.
 
 __ #TODO
 __ #TODO
```

Comment:
What are these for?

Reply:
What do you mean? These are normal link definitions, just not (yet) pointing anywhere useful. Exactly like the "low-level API documentation" one that I filled in.

Reply:
Yes and what are we expecting here in the future? In anyway... has nothing to do with this PR.

Reply:
Just look at the link text in the paragraph right above: For the first one it's `` `high-level API`__ ``, for the second one it's `` `here`__ ``. :)